resources/modes/print/print.js

```diff
@@ -73,6 +84,7 @@ $(function() {
 
       // With nothing in the queue, start autopaint!
       if (cncserver.state.buffer.length === 0) {
+        $("#processing-dialog").show()
```

Comment:
In what way? Have you tested this? Timers/intervals are pretty important for running things so I'd be amazed if out "next tick" loops were causing an issue.